docs/topics/con_key-vertx-concepts.adoc

```diff
@@ -11,14 +11,14 @@ A reactive system, as defined in the link:http://reactivemanifesto.org[reactive
 
 Elastic:: The system remains responsive under varying workload, with individual components scaled and load-balanced as necessary to accommodate the differences in workload. Elastic application deliver the same quality of service regardless of the number of requests they receive at the same time.
 
-Resilient:: The system remains responsive in case any of its individual components fails. Components are isolated from each other, with each individual component being able to quickly recover after failure occurs. Failure of a single component should never prevent other components from functioning beyond a reasonable amount of time. This prevents cascading failure, where the failure of an isolated components causes other components to become blocked and gradually fail.
+Resilient:: The system remains responsive in case any of its individual components fails. Components are isolated from each other, with each individual component being able to quickly recover after failure occurs. Failure of a single component should never prevent other components from functioning in a reasonable amount of time. This prevents cascading failure, where the failure of an isolated components causes other components to become blocked and gradually fail.
 
 Responsive:: Responsive systems are designed to always respond to requests in a reasonable amount of time to ensure a consistent quality of service. To maintain responsiveness, the communication channel between the applications must never be blocked.
 
 Message-Driven:: The individual components of an application use asynchronous message-passing to communicate with each other. The components first detect each other using service discovery. If an event takes place (such as a mouse click or a search query) on a service, the service sends out a message over a common channel (the event bus). The messages is in turn caught and handled by the respective component.
 
 Reactive Systems can be understood as “distributed systems done right”.
-They are designed with asynchrony as their key property in mind.
+They are designed with asynchronous communication as their key property in mind.
```

Comment:
That's also a change in meaning. It's not just that reactive systems communicate asynchronously with other systems, it's also that they are asynchronous inside. You could argue that that's asynchronous communication as well, and you'd technically be right, but I think that would be confusing.

Reply:
All right then, I'll expand on the term to include all the meanings.